### DISCUSSION TITLE
Laravel Slim-skeleton-11.x [bootstrap] directory

### DISCUSSION THREAD

**Discussion by 4m1n0s** [laravel/framework#47835] [Ideas]
Hey,

what do you think about eliminate bootstrap directory

move app.php to app or config folder and add default app provider to withProviders configurations

and cache folder to storage framework cache directory

```php
<?php

use Illuminate\Foundation\Application;
use Illuminate\Foundation\Configuration\Exceptions;
use Illuminate\Foundation\Configuration\Middleware;

/*
|--------------------------------------------------------------------------
| Create The Application
|--------------------------------------------------------------------------
|
| The first thing we will do is create a new Laravel application instance
| which serves as the brain for all of the Laravel components. We will
| also use the application to configure core, foundational behavior.
|
*/

return Application::configure()
    ->withProviders([
        App\Providers\AppServiceProvider::class,
    ])
    ->withRouting(
        web: __DIR__.'/../routes/web.php',
        // api: __DIR__.'/../routes/api.php',
        commands: __DIR__.'/../routes/console.php',
        // channels: __DIR__.'/../routes/channels.php',
    )
    ->withMiddleware(function (Middleware $middleware) {
        //
    })
    ->withExceptions(function (Exceptions $exceptions) {
        //
    })->create();
```

**Comment by henzeb** [comment]
You should place it here or at least refer to it. https://github.com/laravel/framework/pull/47309

**Reply by pxlrbt** [reply]
It's locked, so we can't mention it there.

**Reply by henzeb** [reply]
Because it is already merged. It will be available in laravel 11.

**Reply by mbabker** [reply]
The PRs for the framework and the skeleton repo were always locked and never open to discussion outside of one tweet asking for feedback.  If you've got an idea, the best chance of it getting eyes from the Laravel team is just sending a PR.

**Comment by pxlrbt** [comment]
Just had the same idea to find this discussion. Never changed anything in the bootstrap folder so would be nice to eliminate it.

**Comment by shaedrich** [comment]
Not sure about app.php but for the cache folder, it makes a lot of sense actually.
